scripts/efficientnets/run.py

```diff
@@ -26,28 +26,38 @@
 def main(cfg: DictConfig = None):
     log.info("Training Configs:\n%s", OmegaConf.to_yaml(cfg))
 
+    width, _, img_size, dropout_p, _, _ = compound_params(cfg.name)
+    transforms = T.Compose(
+        [
+            T.Resize(size=(img_size, img_size)),
+            T.ToTensor(),
+            T.Normalize(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)),
+        ]
+    )
     if cfg.pretrained:
-        with torch.set_grad_enabled(False):
-            network = EfficientNet(
-                name=cfg.name,
-                num_classes=cfg.lm.num_classes,
-            ).from_pretrained(name=cfg.name)
+        network = EfficientNet(
```

Comment:
Where have I seen this code, I feel like i am quite familiar with this one 😅😅

Reply:
have other name for that method?

Reply:
I just want to try `classmethod` and end up being almost the same

Reply:
I don't think pytorch need it